callstats-java-sdk/src/main/java/io/callstats/sdk/data/ConferenceStats.java

```diff
@@ -60,31 +60,7 @@ public String getFromUserID() {
 	public void setFromUserID(String fromUserID) {
 		this.fromUserID = fromUserID;
 	}
-	public int getRtt() {
-		return data.getRtt();
-	}
-	public void setRtt(int rtt) {
-		data.setRtt(rtt);
-	}
-	public long getPacketsSent() {
-		return data.getPacketsSent();
-	}
-	public void setPacketsSent(long packetsSent) {
-		data.setPacketsSent(packetsSent);
-	}
-	public long getBytesSent() {
-		return data.getBytesSent();
-	}
-	public void setBytesSent(long bytesSent) {
-		data.setBytesSent(bytesSent);
-	}
-	public double getJitter() {
-		return data.getJitter();
-	}
-	public void setJitter(double jitter) {
-		data.setJitter(jitter);
-	}	
-	
```

Comment:
You are removing lots of things that are declared public. If you are absolutely sure that these are not used, at the very least please upgrade the version to 4.0.0-SNAPSHOT